lib/request.js

```diff
@@ -1,6 +1,7 @@
 'use strict';
 
-const request = require('request');
+// If in browser, use request-browser; else use normal request.
+const request = Window ? require('request-browser') : require('request');
```

Comment:
`typeof Window !== 'undefined' ? ...`

Also, where does request-browser come from?  do we need to update package.json and package-lock.json?